src/index.html

```diff
@@ -17,11 +18,368 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <div class="header__logo">
+        <a href="/">
+          <img
+            src="../src/images/logo.png"
+            alt="Site logo"
+          />
+        </a>
+      </div>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              class="header__link header__link--active is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="header__item"
+            data-qa="nav-hover"
+          >
+            <a
+              class="header__link"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
```

Comment:
The 'data-qa' attribute is correctly set to 'data-qa="card"' for the first card, as required by the task.